### ISSUE TITLE
All about pytorch

### ISSUE THREAD

**Issue by smart022** [smart022/articles#7]
## 参考
1. [pytorch loss function 总结](https://blog.csdn.net/zhangxb35/article/details/72464152)
2. [PyTorch 实践指南 / 如何架构](https://github.com/chenyuntc/pytorch-best-practice)
3. [PyTorch Cookbook（常用代码段整理合集）](https://zhuanlan.zhihu.com/p/59205847)

## 实践
[MNIST Handwritten Digit Recognition in PyTorch](https://nextjournal.com/gkoehler/pytorch-mnist)
[MNIST RNN](https://medium.com/dair-ai/building-rnns-is-fun-with-pytorch-and-google-colab-3903ea9a3a79)
[官方nn example](https://pytorch.org/tutorials/beginner/nn_tutorial.html)

## 问题
[PyTorch 中的 ModuleList 和 Sequential: 区别和使用场景](https://zhuanlan.zhihu.com/p/64990232)

**Comment by smart022** [comment]
# torchtext
[TorchText用法示例及完整代码](https://blog.csdn.net/nlpuser/article/details/88067167)
[torchtext入门教程，轻松玩转文本数据处理](https://zhuanlan.zhihu.com/p/31139113)
[Torchtext 详细介绍 Part.1](https://zhuanlan.zhihu.com/p/37223078)

**Comment by smart022** [comment]
[调参](https://www.cnblogs.com/bamtercelboo/p/7469005.html)
